README.md

```diff
@@ -1,3 +1,55 @@
-# RadCore
+# mas-rad_core
 
-This project rocks and uses MIT-LICENSE.
+This project contains a Rails Engine with shared code used for the Retirement
+Adviser Directory.
+
+![Build Status](https://travis-ci.org/moneyadviceservice/mas-rad_core.svg?branch=master)
+
+## Prerequisites
+
+* [Git](http://git-scm.com)
+* [Ruby 2.2.2](http://www.ruby-lang.org/en)
+* [Rubygems 2.2.2](http://rubygems.org)
+* [Bundler](http://bundler.io)
+* [PostgreSQL](http://www.postgresql.org/)
+* [Elasticsearch >= 1.2](https://www.elastic.co/products/elasticsearch)
+
+## Setup
+
+Clone the repository:
+
+```sh
+$ git clone https://github.com/moneyadviceservice/mas-rad_core.git
+```
+
+Make sure all dependencies are available to the application:
+
+```sh
+$ bundle install
+```
+
+Make sure PostgreSQL is running.
+
+Setup the database:
+
+```sh
+$ cp spec/dummy/config/database.example.yml spec/dummy/config/database.yml
+```
+Be sure to remove or modify the `username` attribute.
+
+```sh
+$ bundle exec rake db:create \
```

Comment:
Note to reviewers: this issue was discussed [here](https://github.com/moneyadviceservice/rad/pull/181/files#r28778203) and we decided to leave as is.